client/containers/DashboardPage/PageDeleteForbidden.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+type Props = {
+	pageData: {
+		title?: string;
+	};
+};
+
+const PageDeleteForbidden = (props: Props) => {
```

Comment:
Personally I would consider merging this functionality with `PageDelete` by adding an `isForbidden` prop to that component and either rendering the existing contents or this new notice based on its value. Otherwise any stylistic changes to that component have to be duplicated into this one.

Reply:
So sensible.
I shall act accordingly.